confgenerator/confgenerator.go

```diff
@@ -31,6 +32,11 @@ type unifiedConfig struct {
 	Metrics collectd.Metrics `yaml:"metrics"`
 }
 
+type unifiedConfigWindows struct {
```

Comment:
I think we should be using the same config format for all platforms.  Can we just generate the OTel config from the existing unifiedConfig struct?

Reply:
Existing UnifiedConfig uses collectd in "yaml:metrics"..i am not sure how i can reuse it. Any ideas?

Reply:
Basically otel metrics and collectd metrics, they use the same interface. So there is only one set of Metrics definition needed. I can send a refactor PR afterwards. WDYT?

Reply:
Sync'ed offline that i will send a follow-up PR for the refactoring